front_end/src/components/landing_page/search_widget/SearchWidget.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import { withStyles } from "@material-ui/core/styles";
+
+import { PropTypes } from "prop-types";
+import { connect } from "react-redux";
+import { Form, Input, FormGroup, Container, Label } from "reactstrap";
+
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import Button from "@material-ui/core/Button";
+import Destination from "./Destination";
+import RoomNumber from "./RoomNumber";
+import CalendarPicker from "./CalendarPicker";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+
+const styles = theme => ({
+  root: {
+    display: "flex",
+    flexWrap: "wrap",
+    opacity: 0.7
+  },
+  button: {
+    margin: theme.spacing.unit,
+    top: 10,
+    height: 40
+  },
+  paper: {
+    // padding: theme.spacing.unit * 2,
+    textAlign: "center",
+    height: 70
+    // color: theme.palette.text.secondary
+  }
+});
+
+class SearchWidget extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      focused: null,
+      name: "hai",
+      labelWidth: 10
+    };
+  }
+  componentDidMount = () => {
+    // during logged in , if we change url to landing/home it will redirect to homepage
```

Comment:
TODO

Reply:
or I guess we can delete it... functionality doesn't change based on authentication